sdk/python/examples/descriptors/nem_namespace.py

```diff
@@ -5,7 +5,6 @@ def descriptor_factory():
 			'type': 'namespace_registration_transaction',
 			'rental_fee_sink': 'TAMESPACEWH4MKFMBCVFERDPOOP4FK7MTDJEYP35',
 			'rental_fee': 50000 * 1000000,
-			'parent_name': '',
```

Comment:
hmm, looking at it again, why was this change needed?

(this is nem, so serialization should result in a difference between UNSET parent_name and EMPTY parent_name)

Reply:
wasn't *needed* 

> this is nem, so serialization should result in a difference between UNSET parent_name and EMPTY parent_name

sdk does not support that difference, so serialization is identical for '' and undefined